webapp/channels/src/components/common/site_name_and_description.tsx

```diff
@@ -7,40 +7,30 @@ import {FormattedMessage} from 'react-intl';
 type Props = {
     customDescriptionText?: string;
     siteName: string;
-}
-
-export default class SiteNameAndDescription extends React.PureComponent<Props> {
-    public static defaultProps: Partial<Props> = {
-        siteName: 'Mattermost',
-    };
+};
 
-    public render(): JSX.Element {
-        const {
-            customDescriptionText,
-            siteName,
-        } = this.props;
-        let description = null;
-        if (customDescriptionText) {
-            description = customDescriptionText;
-        } else {
-            description = (
-                <FormattedMessage
-                    id='web.root.signup_info'
-                    defaultMessage='All team communication in one place, searchable and accessible anywhere'
-                />
-            );
-        }
-
-        return (
-            <React.Fragment>
-                <h1 id='site_name'>{siteName}</h1>
-                <h3
-                    id='site_description'
-                    className='color--light'
-                >
-                    {description}
-                </h3>
-            </React.Fragment>
+export default function SiteNameAndDescription({customDescriptionText, siteName = 'Mattermost'}: Props) {
```

Comment:
Following the same approach used in other places we would use something like:

```suggestion
const SiteNameAndDescription = ({customDescriptionText, siteName = 'Mattermost'}: Props) => {
```

And then add at the end of the file something like `export default SiteNameAndDescription;`.

In fact, in this case because it is a pure component we would add something like `export default React.memo(SiteNameAndDescription);`